src/main/scala/encry/network/DeliveryManager.scala

```diff
@@ -58,6 +81,7 @@ class DeliveryManager extends Actor with StrictLogging {
   }
 
   override def receive: Receive = {
+    case GetStatusTrackerPeer => sender() ! statusTracker.statuses
```

Comment:
I think it's better to move it to the bottom of receive, because it's being used only for test purposes